natgeo/mortar
# Remove unecessary icons

https://jira.nationalgeographic.com/browse/MORTAR-110

Testing instructions

-git checkout 
-gulp serve
-Make sure icons listed in ticket are removed http://localhost:3000/icons
